### DISCUSSION TITLE
Tap on viz dashboard failed

### DISCUSSION THREAD

**Discussion by gxf1027** [linkerd/linkerd2#9269] [Help]
Hi, 
when I click on tap cion on viz dashboard website, a blank page is returned without any information.
However, '_kubectl auth can-i watch deployments.tap.linkerd.io -n emojivoto --as $(whoami)_' returns 'yes', and other features on dashboard is just ok. 
Could you give a help. 
Thanks a lot!

**Comment by gxf1027** [comment]
it works with command line, but it doesn't work with dashboard.

> sudo linkerd viz tap pod/emoji-6c549dc79f-gfbk4 --namespace emojivoto
req id=6:0 proxy=in  src=10.244.1.24:43546 dst=10.244.1.25:8080 tls=true :method=POST :authority=emoji-svc.emojivoto:8080 :path=/emojivoto.v1.EmojiService/ListAll
rsp id=6:0 proxy=in  src=10.244.1.24:43546 dst=10.244.1.25:8080 tls=true :status=200 latency=956µs
end id=6:0 proxy=in  src=10.244.1.24:43546 dst=10.244.1.25:8080 tls=true grpc-status=OK duration=183µs response-length=2140B
req id=6:1 proxy=in  src=10.244.1.24:43546 dst=10.244.1.25:8080 tls=true :method=POST :authority=emoji-svc.emojivoto:8080 :path=/emojivoto.v1.EmojiService/FindByShortcode
rsp id=6:1 proxy=in  src=10.244.1.24:43546 dst=10.244.1.25:8080 tls=true :status=200 latency=356µs

**Reply by kleimkuhler** [reply]
This is likely tracked by #9306.

**Reply by gxf1027** [reply]
> This is likely tracked by #9306.

Thanks a lot. 
When  '2.11.4 stable' is used, tap page works.
